javaparser-core/src/main/java/com/github/javaparser/printer/lexicalpreservation/LexicalPreservingPrinter.java

```diff
@@ -65,6 +67,7 @@
 
     /**
      * Parse the code and setup the LexicalPreservingPrinter.
+     * @deprecated just use the other constructor.
```

Comment:
@ftomassetti - I can't see a good reason to have this constructor, is it okay to deprecate?